src/deployment.h

```diff
@@ -92,17 +99,23 @@ inline bool createAppRunHook(appdir::AppDir &appDir) {
         return false;
     }
 
-    ofs << "# generated by linuxdeploy-plugin-qt" << std::endl
-        << std::endl
-        << "# try to make Qt apps more \"native looking\", if possible" << std::endl
-        << "# see also https://github.com/AppImage/AppImageKit/issues/977#issue-462374883" << std::endl
-        << "# and https://github.com/AppImage/AppImageKit/issues/977#issue-462374883" << std::endl
-        << "case \"${XDG_CURRENT_DESKTOP}\" in" << std::endl
-        << "    *GNOME*|*gnome*|*XFCE*)" << std::endl
-        << "        export QT_QPA_PLATFORMTHEME=gtk2" << std::endl
-        << "        ;;" << std::endl
-        << "esac" << std::endl;
-
+    const bool haveGtk3 = bf::exists(themesPath / "libqgtk3.so");
+
+    ofs <<
+        "# generated by linuxdeploy-plugin-qt\n"
```

Comment:
Please use `std::endl` instead of `\n` to remain consistent with all other file generations.

Reply:
Are you sure? There is no flush needed here. (`std::endl` is a shorthand for `std::flush << '\n'`). The stream will get flushed at scope exit.